gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py

```diff
@@ -27,12 +27,12 @@ def read_alist_file(filename):
 
   myfile    = open(filename,'r')
   data      = myfile.readlines()
-  size      = string.split(data[0])
+  size      = data[0].split()
   numCols   = int(size[0])
   numRows   = int(size[1])
   H = zeros((numRows,numCols))
   for lineNumber in arange(4,4+numCols):
```

Comment:
Several things that would make it easier to understand.
Instead of
```python
from numpy import *
arange(4, 4 + numCols)
```
i suggest
```python
import numpy as np
np.arange(4, 4 + numCols)
```
Star imports almost always discouraged and I just stumbled over the `arange` function. I'd always expect `np.arange`. 
Also, the constant `4` needs some explaining.

Reply:
The star import has been replaced. The constant 4 is because the lines with 1s start in the fifth line of the file.
I added a comment explaining this for now, as the 3 lines in-between are not needed for establishing the sparse matrix.